File: src/targets/php/guzzle/fixtures/full.php
<?php

$client = new \GuzzleHttp\Client();

$response = $client->request('POST', 'http://mockbin.com/har?foo=bar&foo=baz&baz=abc&key=value', [
  'form_params' => [
    'foo' => 'bar'
  ],
  'headers' => [
    'accept' => 'application/json',
    'content-type' => 'application/x-www-form-urlencoded',
    'cookie' => 'foo=bar; bar=baz',
  ],
]);

echo $response->getBody();

File: src/targets/rust/reqwest/fixtures/full.rs
use serde_json::json;
use reqwest;

#[tokio::main]
pub async fn main() {
    let url = "http://mockbin.com/har";

    let querystring = [
        ("foo", "bar,baz"),
        ("baz", "abc"),
        ("key", "value"),
    ];

    let payload = json!({"foo": "bar"});

    let mut headers = reqwest::header::HeaderMap::new();
    headers.insert("cookie", "foo=bar; bar=baz".parse().unwrap());
    headers.insert("accept", "application/json".parse().unwrap());
    headers.insert("content-type", "application/x-www-form-urlencoded".parse().unwrap());

    let client = reqwest::Client::new();
    let response = client.post(url)
        .query(&querystring)
        .headers(headers)
        .form(&payload)
        .send()
        .await;

    let results = response.unwrap()
        .json::<serde_json::Value>()
        .await
        .unwrap();

    dbg!(results);
}
